test/integration/crcsuite/crcsuite.go

```diff
@@ -247,6 +249,48 @@ func CheckOutputMatchWithRetry(retryCount int, retryTime string, command string,
 	return matchErr
 }
 
+// CheckCRCStatus checks that output of status command
+// matches given regex number of consecutive times
+func CheckCRCStatus(state string) error {
+
+	retryDuration, _ := time.ParseDuration("1m")
+	retryCount := 15
+	cmd := "crc status --log-level debug"
+	expression := ``
+	var numConsecutive int
+
+	if state == "Stopped" {
+		numConsecutive = 1
+		expression = `.*Stopped.*`
+	} else {
+		numConsecutive = 3
+		expression = `.*Running \(v\d+\.\d+\.\d+.*\).*`
+	}
```

Comment:
And here the backticks are useful to have because of the `\`

Reply:
I used backticks everywhere because I had to use them here. Will change other places to double quotes.